### PR TITLE
feat: support docker build target

### PR DESCRIPTION
**Pull request by DCamma**

### CODE REVIEW

File: .github/workflows/docker-build-push-ecr.yaml [hunk 1]
```diff
@@ -38,6 +38,10 @@ on:
         description: "Push Image to ECR"
         type: boolean
         default: true
+      docker_target:
+        description: "Build target"
+        type: string
+        required: false
```
Comment (swibrow) [inline .github/workflows/docker-build-push-ecr.yaml:44]: default is false

File: .github/workflows/docker-build.yaml [hunk 2]
```diff
@@ -15,6 +15,11 @@ on:
       artifact_retention_days:
         description: "Number of days to retain the artifact"
         type: number
+      docker_target:
+        description: "Build target"
+        type: string
+        required: false
```
Comment (swibrow) [inline .github/workflows/docker-build.yaml:21]: same here

**Review by swibrow — APPROVED** [review]
All good but I'm not a fan of adding code that is default anyhow.

**Comment by DCamma** [comment]
> All good but I'm not a fan of adding code that is default anyhow.

Missed that, I will remove it

**Pull request merged**